docs/release/expanded/cut_release.md

```diff
@@ -12,7 +12,6 @@
 Example Full Command List (this is not a script!):
 ```
 export SSH_MOUNT_PATH="/var/folders/...krzO/agent.452"
-export GLOBAL_GITCONFIG_PATH="/Users/mtrachier/.gitconfig"
```

Comment:
I see we have more usernames embedded here :/

Reply:
Yes, there were already in the docs... I can open a future PR and rip them out.